src/index.js

```diff
@@ -1,5 +1,29 @@
 import './style.css';
-import addScore from './functions';
+import {
+  addNewGame, getScore, addScore, postScore,
+} from './functions';
 
-const btn = document.querySelector('.submit-button');
-btn.addEventListener('click', addScore);
\ No newline at end of file
+async function consumeAPI() {
+  const gameData = await addNewGame('games/', 'java game');
+  const id = gameData.result.split(':')[1].trim().split(' ')[0];
+  const refresh = document.querySelector('.refresh');
+  const submitBtn = document.querySelector('.submit-button');
+  refresh.addEventListener('click', () => {
+    async function userScores() {
+      const resultScore = await getScore(id);
+      const playersScore = resultScore.result;
+      playersScore.forEach((element) => {
+        addScore(element.user, element.score);
+      });
+    }
+    userScores();
+  });
+  submitBtn.addEventListener('click', () => {
+    const [userName, userScore] = Array.from(document.querySelectorAll('.user-input'));
+    async function saveScore() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = async () => {
     // code here
  }
```